specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -677,43 +656,42 @@ def verify_cell_kzg_proof(commitment_bytes: Bytes48,
 
 ```python
 def verify_cell_kzg_proof_batch(commitments_bytes: Sequence[Bytes48],
-                                column_indices: Sequence[ColumnIndex],
+                                cell_indices: Sequence[CellIndex],
                                 cells: Sequence[Cell],
                                 proofs_bytes: Sequence[Bytes48]) -> bool:
     """
     Verify that a set of cells belong to their corresponding commitments.
 
-    Given four lists representing tuples of (``commitment``, ``column_index``, ``cell``, ``proof``),
+    Given four lists representing tuples of (``commitment``, ``cell_index``, ``cell``, ``proof``),
     the function verifies ``proof`` which shows that ``cell`` are the evaluations of the polynomial
-    associated with ``commitment``, evaluated over the domain specified by ``column_index``.
+    associated with ``commitment``, evaluated over the domain specified by ``cell_index``.
 
     This function implements the universal verification equation that has been introduced here:
     https://ethresear.ch/t/a-universal-verification-equation-for-data-availability-sampling/13240
 
     Public method.
     """
 
-    assert len(commitments_bytes) == len(cells) == len(proofs_bytes) == len(column_indices)
+    assert len(commitments_bytes) == len(cells) == len(proofs_bytes) == len(cell_indices)
     for commitment_bytes in commitments_bytes:
         assert len(commitment_bytes) == BYTES_PER_COMMITMENT
-    for column_index in column_indices:
-        assert column_index < CELLS_PER_EXT_BLOB
+    for cell_index in cell_indices:
+        assert cell_index < CELLS_PER_EXT_BLOB
     for cell in cells:
         assert len(cell) == BYTES_PER_CELL
     for proof_bytes in proofs_bytes:
         assert len(proof_bytes) == BYTES_PER_PROOF
 
     # Create the list of unique commitments we are dealing with
-    deduplicated_commitments = list(set(commitments_bytes))
-    row_commitments = [bytes_to_kzg_commitment(commitment_bytes) for commitment_bytes in deduplicated_commitments]
-    # Create indices list mapping initial commitments (that potentially contains duplicates) to the unique commitments.
-    row_indices = [deduplicated_commitments.index(commitment_bytes) for commitment_bytes in commitments_bytes]
+    unique_commitments = [bytes_to_kzg_commitment(commitment_bytes) for commitment_bytes in set(commitments_bytes)]
```

Comment:
I liked the name `deduplicated_commitments` better, as uniqueness a very strong word. But not a strong opinion here.